### DISCUSSION TITLE
Multiplayer chat should lose focus when pressing esc for the first time instead of quitting the game

### DISCUSSION THREAD

**Discussion by Gonzalo-Bruna** [ppy/osu#15292] [Ideas] [answered]
When using the multiplayer in-game chat, pressing the esc key will quit the game instead of unfocusing the chat.
It is very annoying and it has happened to me many times as well as other users as seen in this screenshot:

![osu_2021-10-25_13-09-31](https://user-images.githubusercontent.com/41755816/138732790-859ceb97-83b4-442d-b12f-a8edc2a9edfb.jpg)

We use esc key to close overlays / delete chat text / pausing the game, so it's natural to press esc to try to unfocus the in-game chat.

The proposal is as simple as to unfocus chat first if it is focused, then quit the game.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15293
